engine/hatchery/swarm/swarm.go

```diff
@@ -601,3 +620,16 @@ func (h *HatcherySwarm) NeedRegistration(m *sdk.Model) bool {
 	}
 	return false
 }
+
+func getContainerNetworks(client *docker.Client, containerId string) ([]string, error) {
+	var cj types.ContainerJSON
+	var err error
+	if cj, err = client.ContainerInspect(context.Background(), containerId); err != nil {
+		return nil, sdk.WrapError(err, "Unable to fetch container info")
+	}
+	nets := make([]string, 0, 4)
+	for name, _ := range cj.NetworkSettings.Networks {
```

Comment:
should omit 2nd value from range; this loop is equivalent to `for name := range ...`